ros/api/v1/api_version.py

```diff
@@ -0,0 +1,14 @@
+from flask_restful import Resource
+from flask import jsonify
```

Comment:
As per design guidelines, standard package import should be on top, 

```
from flask import jsonify
```

Then third party imports 

```
from flask_restful import Resource
from k8s_deployment_status import DeploymentStatus
```

And finally app imports

```
from ros.extensions import metrics
from ros.lib.config import API_VERSION
```
**Note:** These are not formal PEP8 rules, but are common best practices and conventions.